INSTALL.md

```diff
@@ -1,30 +1,70 @@
 # balena CLI Installation Instructions
 
-The easiest and recommended way of installing the CLI on all platforms (Windows, Linux, macOS) is
-to use the [Standalone Installation](#standalone-installation) described below. Some specific CLI
-commands have a few extra installation steps: see section [Additional Dependencies](#additional-dependencies).
+There are 3 options to choose from to install balena's CLI:
+
+* [Executable Installer](#executable-installer): the easiest method, using the traditional
+  graphical desktop application installers for Windows and macOS (coming soon for Linux users too).
+* [Standalone Zip Package](#standalone-zip-package): these are plain zip files with the balena CLI
+  executable in them. Recommended for scripted installation in CI (continuous integration)
+  environments.
+* [NPM Installation](#npm-installation): recommended for developers who may be interested in
+  integrating the balena CLI in their existing Node.js projects or workflow.
+
+Some specific CLI commands have a few extra installation steps: see section [Additional
+Dependencies](#additional-dependencies).
 
 > **Windows users:** We now have a [YouTube video tutorial](https://www.youtube.com/watch?v=2LApclXFqsg)
 for installing and getting started with the balena CLI on Windows!
 
-## Standalone Installation
+## Executable Installer
 
-1. Download the latest zip file for your OS from https://github.com/balena-io/balena-cli/releases.  
-   (Note that "[Darwin](https://en.wikipedia.org/wiki/Darwin_(operating_system))" is the
-   appropriate zip file for macOS.)
-2. Extract the zip file contents to any folder you choose. The extracted contents will include a
-   `balena-cli` folder.
-3. Add the `balena-cli` folder to the system's `PATH` environment variable. See instructions for:
-   [Windows](https://www.computerhope.com/issues/ch000549.htm) |
-   [Linux](https://stackoverflow.com/questions/14637979/how-to-permanently-set-path-on-linux-unix) |
-   [macOS](https://www.architectryan.com/2012/10/02/add-to-the-path-on-mac-os-x-mountain-lion/#.Uydjga1dXDg)
+_Please note: the executable installers are in **beta** status (recently introduced)._
+
+1. Download the latest installer from the [releases page](https://github.com/balena-io/balena-cli/releases).
+   Look for a file name that ends with "installer-BETA", for example:  
+   `balena-cli-v10.13.6-windows-x64-installer-BETA.exe`  
+   `balena-cli-v10.13.6-macOS-x64-installer-BETA.pkg`
```

Comment:
I've made a change here: added the `-BETA` suffix to the native installer file names (the change was to the docs only, as the code was already generating files with the `-BETA` suffix). This is debatable though, maybe we don't want it. I was thinking that the we could remove the `-BETA` suffix when the installers get digitally signed and no longer raise Windows Defender and Apple Gatekeeper warnings of "unknown publisher" or "unidentified developer".